src/Snippets/Docs/Chaos.Outcome.cs

```diff
@@ -105,15 +106,138 @@ public static void OutcomeGeneratorDelegates()
+    }
+
+    public static void AntiPattern_GeneratorDelegateInjectFault()
+    {
+        #region chaos-outcome-anti-pattern-generator-inject-fault
+        var pipeline = new ResiliencePipelineBuilder<HttpResponseMessage>()
+            .AddChaosOutcome(new ChaosOutcomeStrategyOptions<HttpResponseMessage>
+            {
+                InjectionRate = 0.5, // same injection rate for both fault and outcome
+                OutcomeGenerator = static args =>
+                {
+                    Outcome<HttpResponseMessage>? outcome = Random.Shared.Next(350) switch
+                    {
+                        < 100 => Outcome.FromResult(new HttpResponseMessage(HttpStatusCode.InternalServerError)),
+                        < 150 => Outcome.FromResult(new HttpResponseMessage(HttpStatusCode.TooManyRequests)),
+                        < 250 => Outcome.FromResult(CreateResultFromContext(args.Context)),
+                        < 350 => Outcome.FromException<HttpResponseMessage>(new HttpRequestException("Chaos request exception.")),
+                        _ => Outcome.FromResult(new HttpResponseMessage(HttpStatusCode.OK))
+                    };
+
+                    return ValueTask.FromResult(outcome);
+                },
+                OnOutcomeInjected = static args =>
+                {
+                    // You might have to put some logic here to determine what kind of output was injected. 😕
+                    if (args.Outcome.Exception != null)
+                    {
+                        Console.WriteLine($"OnBehaviorInjected, Exception: {args.Outcome.Exception.Message}, Operation: {args.Context.OperationKey}.");
+                    }
+                    else
+                    {
+                        Console.WriteLine($"OnBehaviorInjected, Outcome: {args.Outcome.Result}, Operation: {args.Context.OperationKey}.");
+                    }
+
+                    return default;
+                }
+            })
+            .Build();
+
+        #endregion
+    }
+
+    public static void Pattern_GeneratorDelegateInjectFaultAndOutcome()
+    {
+        #region chaos-outcome-pattern-generator-inject-fault
+        var randomThreshold = Random.Shared.Next(350);
+        var pipeline = new ResiliencePipelineBuilder<HttpResponseMessage>()
+            .AddChaosFault(new ChaosFaultStrategyOptions
+            {
+                InjectionRate = 0.1, // Different injection rate for faults
+                EnabledGenerator = static args => ValueTask.FromResult(ShouldEnableFaults(args.Context)), // Different settings might apply to inject faults
+                FaultGenerator = args =>
+                {
+                    Exception? exception = randomThreshold switch
+                    {
+                        >= 250 and < 350 => new HttpRequestException("Chaos request exception."),
                         _ => null
                     };
 
+                    return new ValueTask<Exception?>(exception);
```

Comment:
It is a minor thing but from my point of view we should have samples with consistent API usage.

Current we have:
```cs
return new ValueTask<Exception?>(exception); //inside FaultGenerator
...
return ValueTask.FromResult(outcome); //inside OutcomeGenerator
```

I would suggest to use either the ctor in both places or the `FromXYZ` methods.